### ISSUE TITLE
Steam Log In Window is Empty (even after installing freetype)

### ISSUE THREAD

**Issue by Tvardovsky** [IsaacMarovitz/Whisky#24]
Steam log in screen appears empty after installing to Windows 10 Bottle.
Everything installed properly.

**Comment by l0stl0rd** [comment]
Same here even tried reinstalling them.

**Comment by BigRyanGames** [comment]
I have the same issue

**Comment by CaioFaSoares** [comment]
Currently with the same issue.
Tried to install dotnet directly, but I only got a blank, white window.

**Comment by TeroKeso** [comment]
In discord some did find a solution not just install freetype, but also all dependings.

Open an x86 terminal session with arch -x86_64 zsh

`/usr/local/bin/brew install zlib freetype sdl2 libgphoto2 faudio jpeg libpng mpg123 libtiff libgsm glib gnutls libusb gettext openssl@1.1 sane-backends`

**Comment by CaioFaSoares** [comment]
Thanks, @TeroKeso 
Tried to install them, but they were already installed.
Ran `/usr/local/brew doctor` and they were unlinked.
Ran `/usr/local/brew link freetype` and it worked.

Thanks again!

**Comment by ElianIsac** [comment]
Did the following steps:

Installed Porting Kit through Homebrew
Made sure freetype was installed
Using the 0.2.0 Version with libraries copied from the porting kit.dmg as stated on the release page
Using the -noreactlogin on steam.exe post install
Reinstalled freetype and all dependencies as TeroKeso suggests
Checked /usr/local/brew doctor, and freettype is already linked
Restarted whisky and created a new bottle multiple times

Still blank screen...

**Comment by CaioFaSoares** [comment]
@ElianIsac What finally made it work here:
- Completely uninstalled both ARM brew and Intel brew, Whisky and Porting Tookit.
- Reinstalled Intel brew
- Downloaded the dmg again.
- Downloaded Whisky again, threw Porting Kit's lib in the folder, created a bottle and threw an old installation Steam.
- Installed brew on 86 shell, installed freetype, linked everything.

**Comment by Tvardovsky** [comment]
Looks like Crossover 22.1 fix is working here 
Drop this file to /Applications/Whisky.app/Contents/Resources/Libraries/Wine/bin
[wineloader64.zip](https://github.com/IsaacMarovitz/Whisky/files/11682229/wineloader64.zip)

**Comment by valyagolev** [comment]
same problem. what worked for me:

1. mount the game porting toolkit dmg, and run
2. 
```
ditto /Volumes/Game\ Porting\ Toolkit-1.0/lib/ `brew --prefix game-porting-toolkit`/lib/
```

3. run not through wine or whisky, but:

`/Volumes/Game\ Porting\ Toolkit-1.0/lib/gameportingtoolkit ~/my-game-prefix ...path/to/steam.exe`

**Comment by serralva-ruben** [comment]
i also get the steam blank login screen, tried everything but nothing seems to work...

**Comment by aayork** [comment]
anyone found a fix to this? ive tried everything in this thread

**Comment by BigRyanGames** [comment]
I had to use the method on applegamingwiki. Installing overwatch now!

**Comment by aayork** [comment]
@BigRyanGames Nice! Mind sending a link my way?

**Comment by BigRyanGames** [comment]
> @BigRyanGames Nice! Mind sending a link my way?
[
](https://www.applegamingwiki.com/wiki/Game_Porting_Toolkit)

it took ages, and is all command line, but it seems to be working.

**Comment by Tvardovsky** [comment]
So it's working for me now.
What I did:
1. Install all dependings as @TeroKeso advised.
2. Delete Whisky with AppCleaner free app including bottles.
3. Install v. 0.2.0 and manually add Game Porting Toolkit
4. Add wineloader64 file (fix for crossover) to binaries.

That's all. Now it shows the login screen correctly and Steam itself.

Bad news: I tested AOE2: DE, Mafia: DE, Prey and L.A. Noire.
AOE2: DE is working well except for fonts in the main menu.
Prey - main menu only, crashes on a "new game".
Mafia and L.A. Noire crashes on start.
